### PR TITLE
Updated Vue Storefront detection

### PR DESCRIPTION
**Pull request by bloodf**
update the Vue Storefront detection, so it can be widely detected.

### CODE REVIEW

File: src/technologies/v.json [hunk 1]
```diff
@@ -488,10 +488,10 @@
     "icon": "vue-storefront.svg",
     "implies": "Vue.js",
     "meta": {
-      "generator": "^Vue Storefront\\ [0-9.]+?$;version=2"
+      "generator": "^Vue Storefront\\ [0-9.]+?$"
```
Comment (AliasIO) [inline src/technologies/v.json:491]: Should we include the version number in this case? I noticed the old code was wrong, it should be:

`"generator": "^Vue Storefront\\ ([0-9.]+)?$\\;version=\\1"`

File: src/technologies/v.json [hunk 2]
```diff
@@ -488,10 +488,10 @@
...
     },
     "scripts": [
-      "(?:\\/([\\d.]+))?\\/vsf-layout;version=1"
+      "vsf-layout(?:-\\w*)?\\.?(?:[0-9a-fA-F]{7,40}|[\\d]+(?:.[\\d]+(?:.[\\d]+)?)?)(?:\\.min)?\\.js"
```
Comment (AliasIO) [inline src/technologies/v.json:494]: Could this simply be `vsf-layout\\;version=1`? Is this used in versions other than 1?

Comment (bloodf) [reply src/technologies/v.json:494]: I tried, but the extension was not recognizing VSF. If I add the version happens the same.

File: src/technologies/v.json (hunk 1 above)
Comment (bloodf) [reply src/technologies/v.json:491]: Adding the version, the extension does not recognize VSF

Comment (AliasIO) [reply src/technologies/v.json:491]: Sorry, the correct syntax is "^Vue Storefront ([0-9.]+)?$\\;version:\\1".

**Comment by AliasIO** [comment]
I updated this in to work with version detection, see:

https://github.com/AliasIO/wappalyzer/commit/2101abddda98ba5d1cc09c806c4f24f6be262c42#diff-34996dcc0286345b8ace13f89663bf0bfdaf8ee97dc72a18dcc3124bd57a062e

**Comment by bloodf** [comment]
@AliasIO the name need to be updated to `Vue Storefront` removing the dash.

Also the current version 6.8.5 is not detecting the integration :(

**Comment by AliasIO** [comment]
I'll update that. It's not working in v6.8.5 because the version syntax was wrong. A new version is currently pending approval, Vue Storefront will be detect correctly in the version after that.

**Pull request closed without merging**